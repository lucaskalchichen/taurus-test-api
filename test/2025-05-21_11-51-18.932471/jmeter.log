2025-05-21 11:51:22,851 INFO o.a.j.u.JMeterUtils: Setting Locale to en
2025-05-21 11:51:22,862 INFO o.a.j.JMeter: Loading user properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/user.properties
2025-05-21 11:51:22,862 INFO o.a.j.JMeter: Loading system properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/system.properties
2025-05-21 11:51:22,862 INFO o.a.j.JMeter: Loading additional properties from: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-51-18.932471/jmeter-bzt.properties
2025-05-21 11:51:22,863 INFO o.a.j.JMeter: Setting System properties from file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-51-18.932471/system.properties
2025-05-21 11:51:22,866 INFO o.a.j.JMeter: Copyright (c) 1998-2022 The Apache Software Foundation
2025-05-21 11:51:22,867 INFO o.a.j.JMeter: Version 5.5
2025-05-21 11:51:22,867 INFO o.a.j.JMeter: java.version=11.0.27
2025-05-21 11:51:22,867 INFO o.a.j.JMeter: java.vm.name=OpenJDK 64-Bit Server VM
2025-05-21 11:51:22,867 INFO o.a.j.JMeter: os.name=Linux
2025-05-21 11:51:22,867 INFO o.a.j.JMeter: os.arch=amd64
2025-05-21 11:51:22,867 INFO o.a.j.JMeter: os.version=5.15.167.4-microsoft-standard-WSL2
2025-05-21 11:51:22,867 INFO o.a.j.JMeter: file.encoding=UTF-8
2025-05-21 11:51:22,867 INFO o.a.j.JMeter: java.awt.headless=true
2025-05-21 11:51:22,867 INFO o.a.j.JMeter: Max memory     =4160749568
2025-05-21 11:51:22,868 INFO o.a.j.JMeter: Available Processors =28
2025-05-21 11:51:22,870 INFO o.a.j.JMeter: Default Locale=English
2025-05-21 11:51:22,871 INFO o.a.j.JMeter: JMeter  Locale=English
2025-05-21 11:51:22,871 INFO o.a.j.JMeter: JMeterHome=/home/lucas/.bzt/jmeter-taurus/5.5
2025-05-21 11:51:22,871 INFO o.a.j.JMeter: user.dir  =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 11:51:22,871 INFO o.a.j.JMeter: PWD       =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 11:51:22,877 INFO o.a.j.JMeter: IP: 127.0.1.1 Name: kalchi-pc FullName: kalchi-pc.
2025-05-21 11:51:22,877 INFO o.a.j.JMeter: user.classpath=/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-51-18.932471:/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-51-18.932471
2025-05-21 11:51:22,877 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-51-18.932471
2025-05-21 11:51:22,879 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-51-18.932471
2025-05-21 11:51:22,884 INFO o.a.j.s.FileServer: Default base='/mnt/d/Universidad/recursos/taurus api/test'
2025-05-21 11:51:22,885 INFO o.a.j.s.FileServer: Set new base='/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-51-18.932471'
2025-05-21 11:51:22,959 INFO o.a.j.s.SaveService: Testplan (JMX) version: 2.2. Testlog (JTL) version: 2.2
2025-05-21 11:51:22,971 INFO o.a.j.s.SaveService: Using SaveService properties version 5.0
2025-05-21 11:51:22,973 INFO o.a.j.s.SaveService: Using SaveService properties file encoding UTF-8
2025-05-21 11:51:22,975 INFO o.a.j.s.SaveService: Loading file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-51-18.932471/modified_requests.jmx
2025-05-21 11:51:22,992 INFO o.a.j.p.h.c.CacheManager: Will only cache the following methods: [GET]
2025-05-21 11:51:22,993 INFO o.a.j.p.h.c.CookieManager: Settings: Delete null: true Check: true Allow variable: true Save: false Prefix: COOKIE_
2025-05-21 11:51:23,001 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/html is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:51:23,001 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xhtml+xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:51:23,001 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:51:23,001 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:51:23,001 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/vnd.wap.wml is org.apache.jmeter.protocol.http.parser.RegexpHTMLParser
2025-05-21 11:51:23,001 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/css is org.apache.jmeter.protocol.http.parser.CssParser
2025-05-21 11:51:23,018 INFO o.a.j.e.StandardJMeterEngine: Running the test!
2025-05-21 11:51:23,018 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 11:51:23,019 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 11:51:23,025 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must contain the string: '.functions.'
2025-05-21 11:51:23,025 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must not contain the string: '.gui.'
2025-05-21 11:51:23,156 INFO o.a.j.JMeter: Running test (1747839083156)
2025-05-21 11:51:23,165 INFO o.a.j.e.StandardJMeterEngine: Starting ThreadGroup: 1 : api-test
2025-05-21 11:51:23,165 INFO o.a.j.e.StandardJMeterEngine: Starting 42 threads for group api-test.
2025-05-21 11:51:23,165 INFO o.a.j.e.StandardJMeterEngine: Thread will continue on error
2025-05-21 11:51:23,165 INFO o.a.j.t.ThreadGroup: Starting thread group... number=1 threads=42 ramp-up=3 delayedStart=false
2025-05-21 11:51:23,170 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-1
2025-05-21 11:51:23,182 INFO o.a.j.p.h.s.HTTPHCAbstractImpl: Local host = kalchi-pc
2025-05-21 11:51:23,182 INFO o.a.j.t.ThreadGroup: Started thread group number 1
2025-05-21 11:51:23,183 INFO o.a.j.e.StandardJMeterEngine: All thread groups have been started
2025-05-21 11:51:23,185 INFO o.a.j.p.h.s.HTTPHC4Impl: HTTP request retry count = 0
2025-05-21 11:51:23,185 INFO o.a.j.s.SampleResult: Note: Sample TimeStamps are START times
2025-05-21 11:51:23,185 INFO o.a.j.s.SampleResult: sampleresult.default.encoding is set to UTF-8
2025-05-21 11:51:23,186 INFO o.a.j.s.SampleResult: sampleresult.useNanoTime=true
2025-05-21 11:51:23,186 INFO o.a.j.s.SampleResult: sampleresult.nanoThreadSleep=5000
2025-05-21 11:51:23,239 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-2
2025-05-21 11:51:23,308 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-3
2025-05-21 11:51:23,380 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-4
2025-05-21 11:51:23,451 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-5
2025-05-21 11:51:23,521 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-6
2025-05-21 11:51:23,594 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-7
2025-05-21 11:51:23,663 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-8
2025-05-21 11:51:23,733 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-9
2025-05-21 11:51:23,804 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-10
2025-05-21 11:51:23,876 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-11
2025-05-21 11:51:23,947 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-12
2025-05-21 11:51:24,020 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-13
2025-05-21 11:51:24,093 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-14
2025-05-21 11:51:24,160 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-15
2025-05-21 11:51:24,231 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-16
2025-05-21 11:51:24,302 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-17
2025-05-21 11:51:24,373 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-18
2025-05-21 11:51:24,445 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-19
2025-05-21 11:51:24,515 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-20
2025-05-21 11:51:24,586 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-21
2025-05-21 11:51:24,656 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-22
2025-05-21 11:51:24,729 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-23
2025-05-21 11:51:24,799 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-24
2025-05-21 11:51:24,869 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-25
2025-05-21 11:51:24,944 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-26
2025-05-21 11:51:25,011 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-27
2025-05-21 11:51:25,082 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-28
2025-05-21 11:51:25,154 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-29
2025-05-21 11:51:25,224 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-30
2025-05-21 11:51:25,288 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-23
2025-05-21 11:51:25,288 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-23
2025-05-21 11:51:25,290 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-28
2025-05-21 11:51:25,290 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-28
2025-05-21 11:51:25,291 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-6
2025-05-21 11:51:25,291 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-6
2025-05-21 11:51:25,291 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-1
2025-05-21 11:51:25,291 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-1
2025-05-21 11:51:25,292 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-21
2025-05-21 11:51:25,292 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-21
2025-05-21 11:51:25,292 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-19
2025-05-21 11:51:25,292 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-19
2025-05-21 11:51:25,293 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-22
2025-05-21 11:51:25,293 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-22
2025-05-21 11:51:25,294 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-30
2025-05-21 11:51:25,294 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-30
2025-05-21 11:51:25,294 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-20
2025-05-21 11:51:25,294 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-20
2025-05-21 11:51:25,295 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-12
2025-05-21 11:51:25,295 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-12
2025-05-21 11:51:25,295 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-3
2025-05-21 11:51:25,295 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-3
2025-05-21 11:51:25,296 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-15
2025-05-21 11:51:25,296 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-15
2025-05-21 11:51:25,296 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-31
2025-05-21 11:51:25,296 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-8
2025-05-21 11:51:25,297 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-8
2025-05-21 11:51:25,297 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-26
2025-05-21 11:51:25,297 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-26
2025-05-21 11:51:25,298 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-9
2025-05-21 11:51:25,298 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-9
2025-05-21 11:51:25,299 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-7
2025-05-21 11:51:25,299 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-7
2025-05-21 11:51:25,300 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-2
2025-05-21 11:51:25,300 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-2
2025-05-21 11:51:25,300 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-27
2025-05-21 11:51:25,300 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-27
2025-05-21 11:51:25,301 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-5
2025-05-21 11:51:25,301 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-5
2025-05-21 11:51:25,301 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-29
2025-05-21 11:51:25,302 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-29
2025-05-21 11:51:25,302 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-24
2025-05-21 11:51:25,302 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-24
2025-05-21 11:51:25,302 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-13
2025-05-21 11:51:25,302 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-13
2025-05-21 11:51:25,303 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-10
2025-05-21 11:51:25,303 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-10
2025-05-21 11:51:25,303 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-4
2025-05-21 11:51:25,303 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-4
2025-05-21 11:51:25,304 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-17
2025-05-21 11:51:25,304 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-17
2025-05-21 11:51:25,304 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-11
2025-05-21 11:51:25,304 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-11
2025-05-21 11:51:25,305 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-25
2025-05-21 11:51:25,305 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-25
2025-05-21 11:51:25,305 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-18
2025-05-21 11:51:25,305 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-18
2025-05-21 11:51:25,306 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-16
2025-05-21 11:51:25,306 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-16
2025-05-21 11:51:25,306 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-14
2025-05-21 11:51:25,306 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-14
2025-05-21 11:51:25,306 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-31
2025-05-21 11:51:25,307 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-31
2025-05-21 11:51:25,366 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-32
2025-05-21 11:51:25,369 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-32
2025-05-21 11:51:25,370 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-32
2025-05-21 11:51:25,437 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-33
2025-05-21 11:51:25,441 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-33
2025-05-21 11:51:25,442 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-33
2025-05-21 11:51:25,508 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-34
2025-05-21 11:51:25,513 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-34
2025-05-21 11:51:25,513 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-34
2025-05-21 11:51:25,580 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-35
2025-05-21 11:51:25,583 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-35
2025-05-21 11:51:25,583 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-35
2025-05-21 11:51:25,650 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-36
2025-05-21 11:51:25,653 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-36
2025-05-21 11:51:25,653 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-36
2025-05-21 11:51:25,721 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-37
2025-05-21 11:51:25,727 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-37
2025-05-21 11:51:25,727 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-37
2025-05-21 11:51:25,793 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-38
2025-05-21 11:51:25,798 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-38
2025-05-21 11:51:25,798 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-38
2025-05-21 11:51:25,863 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-39
2025-05-21 11:51:25,867 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-39
2025-05-21 11:51:25,867 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-39
2025-05-21 11:51:25,934 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-40
2025-05-21 11:51:25,938 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-40
2025-05-21 11:51:25,939 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-40
2025-05-21 11:51:26,006 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-41
2025-05-21 11:51:26,011 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-41
2025-05-21 11:51:26,012 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-41
2025-05-21 11:51:26,076 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-42
2025-05-21 11:51:26,080 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-42
2025-05-21 11:51:26,080 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-42
2025-05-21 11:51:26,081 INFO o.a.j.e.StandardJMeterEngine: Notifying test listeners of end of test
2025-05-21 11:51:26,082 INFO o.a.j.e.StandardJMeterEngine: Forced JVM shutdown requested at end of test
