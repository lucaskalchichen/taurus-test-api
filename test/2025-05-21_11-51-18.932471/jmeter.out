Created the tree successfully using /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-51-18.932471/modified_requests.jmx
Starting standalone test @ 2025 May 21 11:51:23 ART (1747839083014)
Waiting for possible Shutdown/StopTestNow/HeapDump/ThreadDump message on port 4445
Tidying up ...    @ 2025 May 21 11:51:26 ART (1747839086081)
... end of run
